/var/spool/slurmd/job3733375/slurm_script: line 17: cd: /users/aarjun1/users/aarjun1/data/aarjun1/pathtracker/mot/InT-Models-main/: No such file or directory
module: loading 'anaconda/3-5.2.0'
module: anaconda: "See https://docs.ccv.brown.edu/oscar/software/anaconda for potential issues on VNC and batch jobs"
module: unloading 'python/2.7.12'
module: warning: unknown version 'python/3.8.8'
  Please contact CCV support (support@ccv.brown.edu) if you think that software
  with this module name should be installed.
2022-01-01 15:40:57.095363: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /gpfs/runtime/opt/intel/2017.0/lib/intel64:/gpfs/runtime/opt/intel/2017.0/mkl/lib/intel64:/gpfs/runtime/opt/java/8u111/jre/lib/amd64
2022-01-01 15:40:57.095396: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
Fontconfig warning: ignoring UTF-8: not a valid region tag
Traceback (most recent call last):
  File "mainclean.py", line 25, in <module>
    from utils import engine
  File "/gpfs/data/tserre/aarjun1/pathtracker/mot/InT-Models-main/utils/engine.py", line 11, in <module>
    import imageio
ModuleNotFoundError: No module named 'imageio'
